doc/classes/AStar.xml

```diff
@@ -289,6 +289,7 @@
 			</argument>
 			<description>
 				Returns an array with the points that are in the path found by AStar between the given points. The array is ordered from the starting point to the ending point of the path.
+				[b]Note:[/b] This method is not thread-safe. If called from a [Thread], it will return an empty [PackedVector2Array] and will print an error message.
```

Comment:
This should have been `PackedVector3Array` (you can maybe include the fix in another doc PR, e.g. #47772 which needs an amend too).

Reply:
I amended https://github.com/godotengine/godot/pull/47772 to contain this fix.